www/ssc/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                ssc
+version             0.1.55
+categories          www
+platforms           {darwin >= 18} {openbsd >= 7.2}
+supported_archs     x86_64 arm64
+license             GPL-3
+maintainers         ssc.lu:dylan
+description         an opinionated static website nitpicker
+long_description    ${name}, the Static Site Checker, is an opinionated nitpicker for static \
+                    websites, analysing HTML, SVG, MathML, and more, including ontologies. \
+                    It can produce detailed reports, 'repaired' HTML with resolved server \
+                    side includes, site statistics, etc.. Consider it, at best, alpha.
+
+distname            ${name}-v${version}
+distfiles           ssc-v${version}.tgz
+extract.suffix      .tgz
+extract.post_args   "| tar -xzf -"
+
+homepage            https://ssc.lu/
+master_sites        https://github.com/devongarde/ssc/releases/download/v${version}
+
+checksums           rmd160  e33a077b874a567c944533b92b52c7c85a31a3ab \
+                    sha256  a19cd41c7b5bc4d73e6370db61fe47b269dc30b8e7179b417ecd1a3914a3d1be \
+                    size    23013553
+
+depends_lib         port:boost \
+                    port:hunspell \
+                    port:icu \
+                    port:curl
+
+depends_build       port:cmake
```

Comment:
You should not have to specify any explicit dependencies on `cmake` since you have specified the `cmake` portgroup already above.